webpay/api/tests/test_api.py

```diff
@@ -131,6 +132,47 @@ def test_configures_transaction_fail(self):
         res = self.post(req='')  # cause a form error.
         eq_(res.status_code, 400)
 
+    @mock.patch.object(settings, 'PRODUCT_DESCRIPTION_LENGTH', 255)
+    def test_truncate_long_locale_description(self):
+        payjwt = self.payload()
+        payjwt['request']['defaultLocale'] = 'en'
+        payjwt['request']['locales'] = {
+            'it': {
+                'description': 'x' * 256
+            }
+        }
+        req = self.request(payload=payjwt)
+        res = self.post(req=req)
+
+        eq_(res.status_code, 200)
+        req = self.client.session['notes']['pay_request']['request']
+        eq_(len(req['locales']['it']['description']), 255)
+
+    @mock.patch.object(settings, 'PRODUCT_DESCRIPTION_LENGTH', 255)
+    def test_truncate_long_description(self):
+        payjwt = self.payload()
+        payjwt['request']['description'] = 'x' * 256
```

Comment:
While I doubt there's an off-by-one error here it might be possible. Mind making this string a bit longer?